thispastwinter/To-Do
# JKA-21: Initial Theme File With Global Styles Created

## Description

Built a theme.scss that holds styles that will be used globally for easy re-use.

### 📋 Acceptance Criteria

> Checked off by the PR **Assignee(s)**

- [ ] CRITERION_HERE


### 👩‍🔬 Test Instructions

1. Open code and inspect theme.scss.
2. Ensure theme.scss is imported where appropriate and that variables and mixins are called when necessary.

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked


### Additional

> Delete if unneeded

- [ ] Code has been tested and confirmed locally

---

>### Roles & Responsibilities
>
>#### 👨‍💻 Justin Klaas
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻Blake Tarter
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup
